README.md

```diff
@@ -12,14 +14,57 @@ Assuming you already have [Docker](https://www.docker.com/) installed, you can s
 
 ...and view the site locally at: http://localhost:8036/
 
-## Running tests
+## Developing the dashboard
+
+To start development with the dashboard follow the [Getting started](#geting-started) instructions and any changes to the code will be auto built and the browser refreshed.
 
-To run tests locally:
+### To run the tests.
 
 ```
 ./run test
 ```
 
+### Generating a release for Juju
+
+```
+./run exec yarn run generate-release-tarball
+```
+
+### Accessing a lxd bootstrapped Dashboard that's also within a `multipass` VM
+
+When you want to access a dashboard that was automatically installed with a `juju bootstrap` within an lxd in a `multipass` vm from the host.
+
+- [Create a ssh tunnel](#creating-a-tunnel-from-an-lxd-controller-to-a-multipass-host)
+- In the host
+  - `multipass info dev | grep IPv4` and take note of the IP address.
+  - In your browser visit `http://<that ip>:17070`
```

Comment:
got a 'This site can’t be reached' refused to connect. My multipass vm was running properly when I tried